src/components/ProjectDashboard/components/NftRewardsPanel/NftReward/NftReward.tsx

```diff
@@ -59,7 +62,7 @@ export function NftReward({
   return (
     <div
       className={twMerge(
-        'group relative flex h-full w-[252px] cursor-pointer select-none flex-col rounded-lg border border-grey-200 dark:border-slate-500',
+        'group relative flex h-full w-[252px] cursor-pointer select-none flex-col rounded-nft border border-grey-200 shadow-nftCard dark:border-slate-500',
```

Comment:
I'd avoid making a custom config just for nft rounding, you can say `rounded-[10px]` instead for this.

I think you can do the same with shadow as:
`shadow-[0px_4px_14px_0px_rgba(0, 0, 0, 0.0392156862745098)]`
https://tailwindcss.com/docs/drop-shadow#arbitrary-values

Reply:
Yeah good call, done. Spent a while trying to figure out why the shadow class you gave wasn't working and it's because `(0, 0, 0, 0.0392156862745098)` has spaces in it 🤣 😢